src/Microsoft.Net.Http.Headers/EntityTagHeaderValue.cs

```diff
@@ -112,6 +112,24 @@ public override int GetHashCode()
             return _tag.GetHashCode() ^ _isWeak.GetHashCode();
         }
 
+        public static bool CompareStrongly(EntityTagHeaderValue left, EntityTagHeaderValue right)
```

Comment:
Or maybe an enum EntityTagComparison.Strong and EntityTagComparison.Weak. Otherwise it's a little confusing what true and false correspond to.